flocker/common/functional/test_era.py

```diff
@@ -0,0 +1,31 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+
+"""
+Functional tests for ``flocker-node-era`` command.
+"""
+
+from unittest import skipUnless
+from subprocess import check_output
+
+from twisted.python.procutils import which
+
+from .._era import get_era
+from ...testtools import make_script_tests
+
+EXECUTABLE = b"flocker-node-era"
+
+
+class FlockerNodeEraTests(make_script_tests(EXECUTABLE)):
+    """
+    Tests for ``flocker-node-era``.
+    """
+    @skipUnless(which(EXECUTABLE), EXECUTABLE + " not installed")
+    def setUp(self):
+        pass
+
+    def test_output(self):
```

Comment:
This is only supported on Linux, and should be skipped otherwise. Currently causing failing tests: http://build.clusterhq.com/builders/flocker-osx-10.10/builds/6116/steps/trial/logs/problems